includes/core-changes/corefx/2.1/path-apis-dont-throw-exception-for-invalid-paths.md

```diff
@@ -0,0 +1,53 @@
+### Path APIs don't throw an exception for invalid characters
+
+<xref:System.IO.Path?displayProperty=fullName> APIs no longer validate path characters and throw an <xref:System.ArgumentException> if an invalid character is found.
```

Comment:
System.IO.FileInfo was also affected (https://github.com/dotnet/runtime/issues/38487#issuecomment-650665737). I'm not sure what other System.IO APIs were changed.

Reply:
I saw that, but I think it's because it calls GetFilePath.

Reply:
@gewarren Should still be documented (I think). This is an implementation detail that API consumers aren't usually aware of.

Reply:
Okay, I see that the [FileInfo constructor docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.fileinfo.-ctor?view=net-5.0#System_IO_FileInfo__ctor_System_String_) do mention the exception thrown for invalid path characters, so I think you're right. I've updated that exception (and a whole bunch of others that I think are also affected) in https://github.com/dotnet/dotnet-api-docs/pull/6608. I'm wondering if all of those APIs should be mentioned here.